airflow/providers/redis/log/redis_task_handler.py

```diff
@@ -78,7 +78,7 @@ def _read(
         ).decode()
         return log_str, {"end_of_log": True}
 
-    def set_context(self, ti: TaskInstance):
+    def set_context(self, ti: TaskInstance, **kwargs) -> None:
```

Comment:
does this not need `identifier` as param, since we use `inspect` to detect compatibility?

Reply:
the support for identifier has not been added in providers yet(thought to keep of the scope of this PR to just add the functionality to core). I added the **kwargs for the time being to allow static checks to pass which were failing earlier.